.env

```diff
@@ -0,0 +1,6 @@
+GOOGLE_ID='723860365140-n84h96s612vhcf25tf7ic25q7cnnb8fb.apps.googleusercontent.com'
```

Comment:
This file should be in .gitignore

Reply:
It is not ignored by .gitignore for convenience. It would have to be downloaded every time separately when the project is cloned...

Reply:
Exactly. Secrets used for tokens should never reach source control, even though it is a pet project. That is a developer convenience file, so it should be shared among devs using a secure channel.